src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py

```diff
@@ -326,13 +372,14 @@ def content(self):
                 typ=AAZObjectType,
                 typ_kwargs={"flags": {"required": True, "client_flatten": True}}
             )
-            _builder.set_prop("properties", AAZObjectType, typ_kwargs={"flags": {"client_flatten": True}})
+            _builder.set_prop("properties", AAZObjectType, ".", typ_kwargs={"flags": {"required": True, "client_flatten": True}})
 
             properties = _builder.get(".properties")
             if properties is not None:
-                properties.set_prop("advancedDiagnosticConsent", AAZStrType, ".advanced_diagnostic_consent")
+                properties.set_prop("advancedDiagnosticConsent", AAZStrType, ".advanced_diagnostic_consent", typ_kwargs={"flags": {"required": True}})

```

Comment:
Check why this changed as well, looks like its changing the behavior

Reply:
It is true if you look at the spec https://learn.microsoft.com/en-us/rest/api/support/support-tickets/create?view=rest-support-2024-04-01&tabs=HTTP